Snakefile

```diff
@@ -205,7 +229,22 @@ rule host_filter:
                 sample = SAMPLES_SE,
                 run = RUN,
                 end = "SE".split())
-
```

Comment:
This doesn't conform to pep8 style guidelines just yet, but in general you'll want to leave this extra newline